doc/rados/troubleshooting/troubleshooting-mon.rst

```diff
@@ -648,32 +651,34 @@ cluster's configuration files. If Ceph is using the default debug levels, then
 your logs might be missing important information that would help the upstream
 Ceph community address your issue.
 
-To make sure your monitor logs contain relevant information, you can raise
-debug levels. Here we are interested in information from the monitors.  As with
-other components, the monitors have different parts that output their debug
+Raise debug levels to make sure that your Monitor logs contain relevant
+information. Here we are interested in information from the Monitors.  As with
+other components, the Monitors have different parts that output their debug
 information on different subsystems.
 
 If you are an experienced Ceph troubleshooter, we recommend raising the debug
-levels of the most relevant subsystems. Of course, this approach might not be
-easy for beginners. In most cases, however, enough information to address the
-issue will be secured if the following debug levels are entered::
+levels of the most relevant subsystems. This approach might not be easy for
+beginners. In most cases, however, enough information to address the issue will
+be secured if the following debug levels are entered::
 
       debug_mon = 10
       debug_ms = 1
 
 Sometimes these debug levels do not yield enough information. In such cases,
-members of the upstream Ceph community might ask you to make additional changes
+members of the upstream Ceph community will ask you to make additional changes
```

Comment:
S/upstream//

Reply:
See the comment below. TL;DR: I want "upstream" to distinguish the Foundation-led community from the companies that might have their own, pay-for-admission communities.